src/jit/codegen.h

```diff
@@ -15,7 +15,7 @@
 #include "regset.h"
 #include "jitgcinfo.h"
 
-#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_) || defined(_TARGET_ARM_)
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_) || (defined(_TARGET_ARM_)&& !defined(ARM_SOFTFP))
```

Comment:
This change should not be necessary

---
In reply to: [57577636](https://github.com/dotnet/coreclr/pull/3929#discussion_r57577636) [](ancestors = 57577636)